event-management-backend/src/main/java/ro/msg/event_management/controller/TicketController.java

```diff
@@ -108,4 +108,143 @@
         responseBody.put("email", participantEmail);
         return new ResponseEntity<>(responseBody, HttpStatus.OK);
     }
+
+    @PostMapping(value = "/import")
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN')")
+    public ResponseEntity exportTicketsCsv(@RequestParam MultipartFile csv) throws IOException {
+        var inputStream = csv.getInputStream();
+
+        var reader = new BufferedReader(new InputStreamReader(inputStream));
+
+        reader.readLine(); // skip header
+        var errorString = "";
+
+        while(reader.ready()){
+            String line = reader.readLine();
+            var fields = line.split(",");
+
+            var name = fields[0];
+            var email = fields[1];
+            var categoryId = fields[2];
+            var bookingId = fields[3];
+
+            try{
+                Long.parseLong(bookingId);
+                Long.parseLong(categoryId);
+            }
+            catch (NumberFormatException e){
+                errorString += "Id must be number, skipping row\n";
+            }
+            var booking = bookingService.findOne(Long.parseLong(bookingId));
+            var category = categoryService.findOne(Long.parseLong(categoryId));
+
+            if(booking == null){
+                errorString += "Invalid booking, skipping row\n";
+
+            }
+            if(category == null){
+                errorString += "Invalid ticket category, skipping row\n";
+            }
+
+            var ticket = new Ticket();
+
+            if(name.equals("")) {
+                errorString += "Invalid name, skipping row\n";
+                continue;
+            }
+            ticket.setName(name);
+
+            if(email.equals("")) {
+                errorString += "Invalid email, skipping row\n";
+                continue;
+            }
+            ticket.setEmailAddress(email);
+
+            ticket.setTicketCategory(category);
+            ticket.setBooking(booking);
+
+            ticketService.save(ticket);
+        }
+        if(errorString.equals("")){
+            errorString = "Success";
+        }
```

Comment:
Some 'skipping row'-s are missing the 'skip' :).

Same topic, the logic should be moved to TicketService.